src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/checkadddevice_bug1.tt.sarif

```diff
@@ -0,0 +1,2122 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "StaticDriverVersion"
+      },
+      "files": {
+        "..\\..\\..\\rules\\WDM\\CheckAddDevice.slic": {
+          "mimeType": "text/x-cpp"
+        },
+        "file:///d:/slam1/wdk/src_5043/fail_drivers/wdm/fail_driver1/sdv/sdv-harness.c": {
+          "mimeType": "text/x-cpp"
+        }
+      },
+      "results": [
+        {
+          "level": "pass",
+          "message": "The property is satisfied as the driver defines a AddDevice routine.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "..\\..\\..\\rules\\WDM\\CheckAddDevice.slic",
+                "region": {
+                  "startLine": 35
+                }
+              }
+            }
+          ],
+          "codeFlows": [
+            {
+              "locations": [
+                {
+                  "id": 1,
+                  "kind": "call",
+                  "essential": false,
+                  "properties": {
+                    "Step": 0,
+                    "State": "^",
+                    "Caller": "OS",
+                    "Callee": "main"
+                  }
+                },
+                {
+                  "id": 2,
+                  "message": "====Auto=====",
+                  "kind": "continuation",
+                  "essential": false,
+                  "properties": {
+                    "Step": 1,
+                    "State": "^====Auto====="
+                  }
+                },
+                {
+                  "id": 3,
+                  "message": "====Auto=====,====Pre=====,____true",
+                  "kind": "call",
+                  "essential": false,
+                  "properties": {
+                    "Step": 2,
+                    "State": "^====Auto=====^====Pre=====^____true",
```

Comment:
I actually convert the state ^ to a comma and delete the leading ^. We need a better message. Will address this once we look at the actual SDV viewer.